### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dmitruz**
- [DEMO LINK](https://dmitruz.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://dmitruz.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,8 +9,25 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <script src="chrome-extension://mooikfkahbdckldjjndioackbalphokd/assets/prompt.js"></script>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="search__form"
+    data-qa="big">
```
Comment (volodymyr-soltys97) [inline src/index.html:16]: Need to fix code style everywhere
```suggestion
    <form
      action="#"
      class="search__form"
      data-qa="big"
    >
```

File: src/index.html [hunk 2]
```diff
@@ -9,8 +9,25 @@
...
+      <label class="search__form--label">
+        <span class="search__form--image"></span>
```
Comment (volodymyr-soltys97) [inline src/index.html:18]: The tag span is redundant here, need to add image as a background
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Comment by dmitruz** [comment]
already fixed. Please check.

File: src/index.html [hunk 3]
```diff
@@ -9,8 +9,23 @@
...
+    <script src="chrome-extension://mooikfkahbdckldjjndioackbalphokd/assets/prompt.js"></script>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="form"
+       data-qa="big">
+      <label class="label">
+        <input type="text" class="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
```
Comment (etojeDenys) [inline src/index.html:20]:
```suggestion
        <input 
          type="text" 
          class="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

File: src/index.html [hunk 4]
```diff
@@ -9,8 +9,23 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <script src="chrome-extension://mooikfkahbdckldjjndioackbalphokd/assets/prompt.js"></script>
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#" class="form"
+       data-qa="big">
```
Comment (etojeDenys) [inline src/index.html:16]: same here

**Review by etojeDenys — APPROVED** [review]
imageyou can read about it in the checklist to this task

File: src/index.html (hunk 4 above)
Comment (dmitruz) [reply src/index.html]: done

**Pull request closed without merging**